CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.0] - 2019-01-18
+### Unreleased
```

Comment:
This should be:

```suggestion
### Added
```

Reply:
Got it. I thought I only need to change this during the gem preparation for deployment.

Reply:
That's right, but it's the one above:

```
## [Unreleased] #--- this one
### Added
- Something added

### Changed
- Something changed
```

Reply:
We'll release this version immediately though so no need to make it `[Unreleased]`